SourceX/DiabloUI/selhero.cpp

```diff
@@ -370,4 +375,53 @@ BOOL UiSelHeroMultDialog(
 	selhero_isMultiPlayer = true;
 	return UiSelHeroDialog(fninfo, fncreate, fnstats, fnremove, dlgresult, name);
 }
+
+const char *selhero_GenerateName(std::uint8_t hero_class)
+{
+	// TODO: Replace these with lore-appropriate names. Currently they're generic
+	// fantasy names.
+	static const char *const kNames[3][10] = {
+		{
+		    "Arth",
+		    "Blulcorth",
+		    "Eadweard",
+		    "Eadwulf",
+		    "Ferthun",
+		    "Gurstudd",
+		    "Nehlem",
+		    "Tulvim",
+		    "Uriel",
+		    "Walmund",
+		},
+		{
+		    "Arleta",

```

Comment:
Here is a better list of rouge names:
```
Moreina 
Akara
Kashya
Flavie
Divo
Oriana
Iantha
Shikha
Basanti
Elexa
```

Moreina is the official name of the D1 rouge. Akara, Kashya, and Flavie are said to be her guild mates and appear in D2. The rest hare names of rouge hirlings from D2.

Reply:
Done